src/platform/datapath_raw_socket.c

```diff
@@ -17,20 +17,36 @@
 #pragma warning(disable:4116) // unnamed type definition in parentheses
 #pragma warning(disable:4100) // unreferenced formal parameter
 
+
+uint32_t
+CxPlatGetRawSocketSize () {
```

Comment:
```suggestion
CxPlatGetRawSocketSize(void) {
```
nit: We don't add spaces, and for `c` code we also add `void` instead empty functions.